packages/react/src/components/Authenticator/FederatedIdentity/types.ts

```diff
@@ -0,0 +1,72 @@
+import { FederatedIdentityElements } from './context/elements';
+import React from 'react';
+import { IdentitiesControl } from './controls/IdentitiesControl';
+
+export type socialProvidersUnion = 'amazon' | 'apple' | 'facebook' | 'google';
+export type AuthProvider = 'Amazon' | 'Apple' | 'Facebook' | 'Google';
+
+export const socialProviderList = ['amazon', 'facebook', 'apple', 'google'];
```

Comment:
Think these are probably already available somewhere in the _ui-react-core_ or _ui_ packages